docs/source/additional_features.rst

```diff
@@ -72,10 +72,10 @@ and :pep:`557`.
 Caveats/Known Issues
 ====================
 
-Some functions in the ``dataclasses`` module, such as ``replace()`` and ``asdict()``,
+Some functions in the :py:mod:`dataclasses` module, such as :py:func:`~dataclasses.replace` and :py:func:`~dataclasses.asdict`,
 have imprecise (too permissive) types. This will be fixed in future releases.
 
-Mypy does not yet recognize aliases of ``dataclasses.dataclass``, and will
+Mypy does not yet recognize aliases of :py:func:`dataclasses.dataclass`, and will
```

Comment:
This one causes is a regression in rendering IMO. The `:py:func:` version gets rendered with a trailing `()` pair, while the original gets rendered without it. Since an alias to a function does not involve calling it, IMO it should be rendered without `()`. (Maybe one of the other `:py:` things link the same way but render without `()`?)

Reply:
Dang, missed that. Sure, the proper ref here would be
```
:py:func:`dataclasses.dataclass <dataclasses.dataclass>`
```
Will fix that!